_data/beps_completed.yml

```diff
@@ -13,13 +13,19 @@
     - name: Kirstie Whitaker
   content:
     - raw
+  google_doc_created:
+  pull_request_created: 2020-06
+  pull_request_merged: 2021-02
 
 - number: "003"
   title: Common Derivatives
   leads:
     - name: Chris Markiewicz
```

Comment:
Is the goal here to be latest or comprehensive? Chris Gorgolewski was definitely lead on this until he left.

Reply:
maybe nicer to be comprehensive, no?
I can think of a couple of other BEP where the baton was passed from lead to another.

Reply:
I would prefer to include historical leads. We could add `(former)` if the BEP is still open and they are no longer leading it.